cidemipyme-gatsby/src/components/sections/WorkSection.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import Link from 'gatsby-link';
+import PropTypes from 'prop-types';
+import {
+  Container,
+  Row,
+  Col,
+  Card,
+  CardImg,
+  CardTitle,
+  CardText,
+  CardDeck,
+  CardBody,
+  CardHeader,
+  CardColumns,
+} from 'reactstrap';
+import cide_fb_1 from '../../assets/images/cide_fb_1.jpg';
```

Comment:
You can use this `https://webpack.js.org/configuration/resolve/#resolve-alias` to avoid doing relative imports, just create an alias for the root folder